runtime/config/manager_internal.go

```diff
@@ -0,0 +1,99 @@
+package config
+
+import (
+	"encoding/base64"
+	"fmt"
+	"os"
+	"sync"
+	"sync/atomic"
+	"testing"
+
+	jsoniter "github.com/json-iterator/go"
+
+	"encore.dev/appruntime/model"
+	"encore.dev/appruntime/reqtrack"
+)
+
+type ValueID uint64
+type ValuePath = []string
```

Comment:
Do we need this to be an alias for some reason? In general making things different types is clearer if it's warranted to have a named type in the first place